stable/ksoc-plugins/templates/openshift.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if and .Values.openshift .Values.openshift.enabled -}}
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: ksoc-scc
```

Comment:
RAD?

Reply:
There is nothing constraining us to use ksoc. I changed it to rad